Packs/ctf01/Integrations/CortexXDRIRCTF/CortexXDRIRCTF.py

```diff
@@ -397,104 +397,6 @@ def get_tenant_info_command(client: Client):
     )
 
 
-def get_incidents_command(client, args):
```

Comment:
It has get_incidents implemented in its Client object, so I believe if it is configured right, it should perform a real API call. 
Since the client object we are passing is the one implemented in CortexXDRIRCTF (which inherits from CoreClient), any client operations will be the same as before, and removing the code here should just reduce the code duplication.

Reply:
but TBH the xdr-get-incidents command is in the code but not in the yml, so I think it's not even exposed to the user